core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -114,6 +136,59 @@ oppia.factory('UrlInterpolationService', ['alertsService',
         match = filledUrl.match(INTERPOLATION_VARIABLE_REGEX);
       }
       return filledUrl;
+    },
+
+    /**
+     * Given an resource path, returns the relative url path to that resource
+     * prefixing the appropriate cache_slug to it.
+     */
+    getStaticResourceUrl: function(resourcePath) {
+      return getCachePrefixedUrl(resourcePath);
+    },
+
+    /**
+     * Given an image path, returns the complete url path to that image
+     * prefixing the appropriate cache_slug to it.
+     */
+    getStaticImageUrl: function(imagePath) {
+      validateResourcePath(imagePath);
+      return getCachePrefixedUrl('/assets/images' + imagePath);
+    },
+
+    /**
+     * Given an extension resource path, returns the complete url path to that
+     * extension resource, prefixing the appropriate cache_slug to it.
+     */
+    getExtensionResourceUrl: function(extensionResourcePath) {
+      return getCachePrefixedUrl(extensionResourcePath);
```

Comment:
Actually, the images might be a separate issue, but then my question would change to: why do we need a separate method here?